src/moscot/datasets.py

```diff
@@ -171,6 +180,40 @@ def zebrafish(
     return adata, trees
 
 
+def bone_marrow(
+    path: PathLike = "~/.cache/moscot/bone_marrow.h5ad",
+    *,
+    rna: bool,
+    **kwargs: Any,
+) -> AnnData:
+    """Multiome data of bone marrow measurements :cite:`luecken:21`.
+
+    Contains processed counts of 6,224 cells. The RNA data was filtered to 2,000 top
+    highly variable genes, the ATAC data was filtered to 8,000 top highly variable
+    peaks.
+
+    Parameters
+    ----------
+    path
+        Path where to save the file.
+    rna
+        Returns the RNA data if `True`, otherwise returns ATAC data.
+    kwargs
+        Keyword arguments for :func:`scanpy.read`.
+
+    Returns
+    -------
+    Annotated data object.
+    """
+    if rna:
+        return _load_dataset_from_url(
+            path, backup_url="https://figshare.com/ndownloader/files/40046950", expected_shape=(6224, 2000), **kwargs
```

Comment:
Path needs to postfixed with something, otherwise you will get shape mismatch when loading from cache.